collections/_posts/2024-04-18-DehazingDiffusion.md

```diff
@@ -0,0 +1,380 @@
+is the perturbation kernel of the diffusion process. With a sufficiently large dataset, DSM enables approximating the true score
+$$s_{\theta}(x_t,t) \simeq \nabla_{x_t} \log p(x_t).$$
+
+Then, during inference the learned score function is substituted in the reverse-time SDE and the reverse-time diffusion process is discretized in a series of time steps and solved with the Euler-Maruyama method. The update rule is given by:
+
+$$ x_{t-\Delta t} = x_t - \sigma^{2t} s_{\theta}(x_t)\Delta t +\sigma^t \sqrt{|\Delta t|}z, $$
+
+where
+$$z \sim N(0,1).$$
+The score-based denoising term moves the solution towards
+$$p(x)$$
+and the second term corrupts it again to correct errors made in earlier sampling steps to prevent solutions to converge solely to high-density regions.
+
+(For more details, look at the [tutorial on score-based diffusion models](https://creatis-myriad.github.io./tutorials/2023-05-09-tutorial-score-based-models.html).)
+
+<br/>
+
+## Proposed dehazing method
+The authors tailored several techniques used in deep learning and diffusion models to effciently apply them to ultrasound data.
+
+<u>Joint posterior sampling</u>
+
+The idea is to combine two prior distributions during posterior sampling by sampling from them in parallel whilst conditioning on the measurement.
+
+The joint posterior sampling
+$$p_{X|Y}(x,h|y)$$
+is achieved through the formulation of a **joint conditional diffusion process**
+$$\{x_t,h_t|y\}_{t \in [0,1]},$$
+that gives a joint conditional reverse-time SDE:
+
+$$ d(x_t,h_t) = - \sigma^{2t} \nabla_{x_t, h_t} \log p(x_t,h_t|y)dt + \sigma^t dw_t. $$
+
+From the Bayesian formulation presented at the beginning, we can construct two separate diffusion processes defined by two different score models:
+$$s_{\theta}(x_t,t) \simeq \nabla_{x_t} \log p(x_t)$$
+and
+$$s_{\psi}(h_t,t) \simeq \nabla_{h_t} \log p(h_t).$$
+The gradients of the posterior with respect to
+$$x$$
+and
+$$h$$
+are:
+
+$$ \begin{equation*}
+\begin{cases}
+	\nabla_{x_t} \log p(x_t,h_t|y) \approx s^*_{\theta}(x_t,t) + \nabla_{x_t} \log p(y|x_t,h_t) \\
+	\nabla_{h_t} \log p(x_t,h_t|y) \approx s^*_{\psi}(h_t,t) + \nabla_{h_t} \log p(y|x_t,h_t) 
+\end{cases}
+\end{equation*} $$
+
+Then, substituting the posterior in the joint conditional reverse-time SDE allows to sample from the posterior and eventually obtain both the dehazed signal
+$$x$$
+and the haze estimation
+$$h.$$
+
+Finally, the likelihood
+$$p(y|x_t,h_t)$$
+is usually intractable and has to be approximated. A solution is to corrupt the measurement
+$$y$$
+along the diffusion process
+$$\{y_t\}_{t\in[0,1]}.$$
+We obtain the projected value
+$$ \hat{y}_t \sim q(y_t|y_0), $$
+which is an approximation of the noise-perturbed likelihood:
+
+$$ p(y|x_t,h_t) \approx p(\hat{y}_t|x_t,h_t) \sim N(x_t + h_t, \rho^2I). $$
+
+<u>Learning ultrasound priors</u>
+
+Usually, generative models are used for applications in the image domain. In the case of ultrasound signals, some considerations need to be made.
```

Comment:
Phrasing:
```suggestion
Usually, generative models are used for applications in the image domain. In the case of ultrasound signals, some considerations need to taken into account.
```

Reply:
i think you forgot a "be" before taken. i did a commit with the proposed changes